rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1349,6 +1373,21 @@ public Sha256Hash getBtcBlockchainBlockHashAtDepth(int depth) throws BlockStoreE
         return blockAtDepth.getHeader().getHash();
     }
 
+    public byte[] getBtcBlockchainBlockHeaderByHeight(int height) throws BlockStoreException, IOException {
```

Comment:
Wasn't this the conflicting method that forced limiting the receiveHeader method?

This method seems exploitable to me. There's a map of `height -> block hash` that can be used here

Reply:
Thanks! Done